esignet-core/src/main/java/io/mosip/esignet/core/constants/Constants.java

```diff
@@ -53,8 +53,11 @@ public class Constants {
     public static final String VERIFICATION_COMPLETE = "COMPLETED";
     public static final String VERIFIED_CLAIMS = "verified_claims";
     public static final String PAR_CACHE = "par";
+
+    // client additionalConfig
     public static final String USERINFO_RESPONSE_TYPE = "userinfo_response_type";
     public static final String CONSENT_EXPIRE_IN_MINS = "consent_expire_in_mins";
+    public static final String DPOP_CONFIG_KEY = "dpop_bound_access_tokens";
```

Comment:
We should Name the key after the config, it should be  DPOP_BOUND_ACCESS_TOKENS = "dpop_bound_access_tokens"; pls notice the previous key naming convention.